src/Test/GetTargets.hs

```diff
@@ -0,0 +1,28 @@
+module Test.GetTargets(main) where
+
+import Control.Monad
+import Development.Shake
+import Test.Type
```

Comment:
FWIW this import is currently redundant, and is why all the tests fail. However, with my suggestion below it would be used once more.